php-classes/Emergence/People/InvitationsRequestHandler.php

```diff
@@ -163,7 +173,7 @@ public static function handleAcceptRequest()
 
                 // promote person to user and set password
                 if ($Recipient->Class == Person::class) {
-                    $Recipient = $Recipient->changeClass(User::class);
+                    $Recipient = $Recipient->changeClass($Invitation->UserClass ?: User::getDefaultClass());
```

Comment:
@nbey I don't think this is needed, we want to maintain backwards compatibility for API clients where possible but I don't think we should code around migrations not being run that are committed with the changes. Things work without this right?